### PR TITLE
Promote @songy23 to approver role

### LINKED ISSUE #7271: Track @songy23 interest and contributions for approver
This is an issue to track my (@songy23) contributions to the collector repo and track when I meet the requirements to (re-)become approver. I am interested and have time commitment to contribute to the collector repo.

#### Core repo
Screenshot 2023-03-01 at 10 54 03 AM

- [76 commits](https://github.com/open-telemetry/opentelemetry-collector/commits?author=songy23)
- [9 issues](https://github.com/open-telemetry/opentelemetry-collector/issues?q=+is%3Aissue+author%3Asongy23)
- [27 PRs authored](https://github.com/open-telemetry/opentelemetry-collector/pulls?q=+is%3Apr+author%3Asongy23)
- [194 PRs reviewed](https://github.com/open-telemetry/opentelemetry-collector/pulls?q=+is%3Apr+reviewed-by%3Asongy23+)

#### Contrib repo
Screenshot 2023-03-01 at 10 59 09 AM

- [106 commits](https://github.com/open-telemetry/opentelemetry-collector-contrib/commits?author=songy23)
- [5 issues](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=+is%3Aissue+author%3Asongy23)
- [26 PRs authored](https://github.com/open-telemetry/opentelemetry-collector-contrib/pulls?q=+is%3Apr+author%3Asongy23)
- [26 PRs reviewed](https://github.com/open-telemetry/opentelemetry-collector-contrib/pulls?q=+is%3Apr+reviewed-by%3Asongy23+)

#### Emeritus Approver

I'm listed under the [Emeritus Approvers in the collector repo](https://github.com/open-telemetry/opentelemetry-collector#contributing), and am following this section: https://github.com/open-telemetry/community/blob/main/community-membership.md#emeritus-maintainerapprovertriager:

> When a contributor returns to being more active, they may be promoted back to a their previous position at the discretion of the current maintainers following the process defined above.

### PR DESCRIPTION
**Pull request by mx-psi**
Fixes open-telemetry/opentelemetry-collector#7271

### CODE REVIEW

**Review by bogdandrutu — APPROVED** [review]

**Review by songy23 — APPROVED** [review]
Thank you! 🚀 🚀

**Review by codeboten — APPROVED** [review]

**Review by dmitryax — APPROVED** [review]

**Comment by mx-psi** [comment]
Will wait until tomorrow to merge to give one more day for reviews

**Review by evan-bradley — APPROVED** [review]

**Pull request merged**
